functions/src/modules/SE.functions.ts

```diff
@@ -50,11 +47,11 @@ export const uploadProcessing = functions.storage.object()
     // 2. Update the Task
     let taskRestorationUpdate = {
         status: 'Uploaded',
-        timestampLastVersion: Math.round((new Date()).getTime() / 1000)
+        timestampLastVersion: admin.database.ServerValue.TIMESTAMP
     };
 
     if (!task.restoration.timestampFirstVersion)
-        taskRestorationUpdate['timestampFirstVersion'] = Math.round((new Date()).getTime() / 1000);
+        taskRestorationUpdate['timestampFirstVersion'] = admin.database.ServerValue.TIMESTAMP;
```

Comment:
Can't we use a property assignment syntax here?

Reply:
It's not a property of the **taskRestorationUpdate** object, so the compiler would complain if used  as a property.
Also I'm adding it only the first time the `restoration` sub-node is updated .. so I can't add a  placeholder property of the same name to the **taskRestorationUpdate** object.